code/modules/vehicles/wheelchair.dm

```diff
@@ -117,3 +128,9 @@
 	var/obj/vehicle/ridden/wheelchair/wheelchair_unfolded = new unfolded_type(location)
 	wheelchair_unfolded.add_fingerprint(user)
 	qdel(src)
+
+/obj/vehicle/ridden/wheelchair/proc/attach_bell(obj/structure/desk_bell/bell)
+	bell_attached = bell
+	src.generate_actions()
+	add_overlay("wheelchair_bell")
+	desc += "There is a small bell attached to the handle."
```

Comment:
This feels like it wouldn't have a space. You should override `examine()` instead.